kong/runloop/balancer.lua

```diff
@@ -224,8 +224,18 @@ do
       elseif action == "removed" then
         local ok, err = healthchecker:remove_target(ip, port)
         if not ok then
-          log(ERR, "[healthchecks] failed adding a target: ", err)
+          log(ERR, "[healthchecks] failed removing a target: ", err)
+        end
+
+      elseif action == "health" then
+        local health_status
```

Comment:
can we change the variable name to `balancer_status` that's more appropriate imo